### ISSUE TITLE
investigating "map to..." cases of the keyword vocab.

### ISSUE THREAD

**Issue by laureD19** [SSHOC/marketplace-curation#16]
see https://docs.google.com/spreadsheets/d/1-Oh9_SxIhfMAT6KNJrMf4LetCpy5s1fHZEyTL__TUVA/edit#gid=883102613

investigating what should be the process to manage the "map to.. [other vocabularies]" cases of the keywords.
(except for the "map to standard vocabulary" assigned to Klaus. See #18 

If mapping to other concept in existing vocabulary found, create a new line in the ["mappings" tab of the spreadsheet](https://docs.google.com/spreadsheets/d/1-Oh9_SxIhfMAT6KNJrMf4LetCpy5s1fHZEyTL__TUVA/edit#gid=0).

**Comment by laureD19** [comment]
Once lines added to the mappings tab - run again https://github.com/SSHOC/marketplace-curation/blob/dev/5.1CurationKeywords.ipynb

**Comment by carikan** [comment]
I went through the keyword list and checked if they exist in vocabs by using [sshompvocabsearch]( https://github.com/mkrzmr/sshompvocabsearch/tree/main ) that @mkrzmr created.
If a keyword existed in a vocabulary I mapped them to the existing vocabulary URL. 
If there are two vocabulary entries for the same term like in the case of "economics" which is an upper category with subgategory, I preffered the upper category as @KlausIllmayer suggested. 
The keyword list can be sorted according to ID and the newly added keywords can be checked again via vocabsearch.
